apps/src/templates/codeReviewGroups/CodeReviewGroupsDataApi.js

```diff
@@ -52,6 +63,17 @@ export default class CodeReviewGroupsDataApi {
     return groups;
   };
 
+  addDroppableIdToGroups = groups => {
```

Comment:
mmm yeah why did I move this here...I think it could no longer go in `CodeReviewGroupsManager` b/c group state is now being managed further up. I could move to `ManageCodeReviewGroups` (which still feels weird) but maybe less weird than putting it here?